README.md

```diff
@@ -1,5 +1,27 @@
+# PMEM-CSI for Kubernetes
+
+<h2>*Note: This is Alpha code and not production ready.*</h2>
+
+Intel PMEM-CSI is a storage driver for container orchestrators like
+Kubernetes. It makes local persistent memory
+([PMEM](https://pmem.io/)) available as a filesystem volume to
+container applications. It can currently utilize non-volatile memory
+devices that can be controlled via the [libndctl utility
+library](https://github.com/pmem/ndctl). In this readme, we use
+*persistent memory* to refer to a non-volatile dual in-line memory
+module (NVDIMM).
+
+The [v0.6.0release](https://github.com/intel/pmem-csi/releases/tag/v0.6.0)
+is the latest feature release and is [regularly updated](./DEVELOPMENT.md#release-management) with newer base images
+and bug fixes. Older versions are no longer supported.
+
+The PMEM-CSI driver follows the [CSI
+specification](https://github.com/container-storage-interface/spec) by
+listening for API requests and provisioning volumes accordingly.
+
+<h3>Table of Contents</h3>
```

Comment:
Can we avoid HTML tags like `<h3>` in favor of normal markdown? Both works of course, markdown would just be more consistent because that's what we use for the other sections.

Reply:
Yep.